code/game/objects/items/mountable_frames/sound_system.dm

```diff
@@ -0,0 +1,12 @@
+/obj/item/mounted/frame/soundsystem
+	name = "sound system frame"
+	desc = "Used for repairing or building sound systems"
+	icon = 'icons/obj/radio.dmi'
+	icon_state = "wallradio"
+	flags = FPRINT
+	w_type=2*RECYK_METAL
+	mount_reqs = list("nospace", "simfloor")
+

```

Comment:
Yeah and I tested it and one of the sound systems didn't respond at all when there were two